### ISSUE TITLE
fhzero configurations in GEFSv13 prototype run

### ISSUE THREAD

**Issue by junwang-noaa** [ufs-community/ufs-weather-model#2359]
## Description

The atmosphere accumulation bucket was updated in FV3TAM PR [#835](https://github.com/NOAA-EMC/fv3atm/pull/835). The bucket in GEFSv13 restart run looks not correct, further investigation is needed to make the feature work for GEFSv13.

**Comment by junwang-noaa** [comment]
GEFSv13 run directory: /lfs/h2/emc/gefstemp/Bing.Fu/fcst.75320

**Comment by lisa-bengtsson** [comment]
Is this a new issue in all applications or just the ensemble configuration?

**Comment by junwang-noaa** [comment]
@lisa-bengtsson No, we are looking at the test case when GEFSv13 restarts at fh=03 (not a multiple of fhzeros) and has bucket of 3hr for fh=3-6hr and 6hr for the rest of forecast time. I don't think other tests use his configuration.

**Comment by junwang-noaa** [comment]
@bingfu-noaa, I checked the run output in /lfs/h2/emc/gefstemp/Bing.Fu/fcst.75320. In the run output file, I saw:

   fhzero            :    3.00000000000000      
  fhzero_array      :    3.00000000000000        6.00000000000000
  fhzero_fhour      :    6.00000000000000       0.000000000000000E+000

Would you please change the 2nd fhzero_fhour to a large number, such as:
&gfs_physics_nml
  fhzero       = 6
  fhzero_array = 3,6
  fhzero_fhour = 6, **2400**

The fhzero will keep having  value 3 if the second element of fhzero_fhour is zero. With the change, we have:

at the first step:
```
 ---isec,seconds         300       11100
   gfs diags time since last bucket empty:   8.333333333333333E-002 hrs
 in atmos_model update, fhzero=   3.00000000000000      fhour=   3.083333
   3.000000
```
at fh=06:
```
 ---isec,seconds       10800       21600
  gfs diags time since last bucket empty:    3.00000000000000      hrs
 in atmos_model update, fhzero=   3.00000000000000      fhour=   6.000000
   6.000000
```
at fh=09:
```
 ---isec,seconds       10800       32400
  gfs diags time since last bucket empty:    3.00000000000000      hrs
 in atmos_model update, fhzero=   6.00000000000000      fhour=   9.000000
   6.000000
```
at fh=12
```
---isec,seconds       21600       43200
  gfs diags time since last bucket empty:    6.00000000000000      hrs
 in atmos_model update, fhzero=   6.00000000000000      fhour=   12.00000
   12.00000
```

**Comment by bingfu-NOAA** [comment]
@lisa-bengtsson please check out.old to see the original log file. The log file you saw are from some tests I made, please ignore it.

**Comment by bingfu-NOAA** [comment]
@junwang-noaa I will follow your suggestion to take another test. Do you think this can be resolved by setting specific values for the newly-added fhzero* parameters?

**Comment by junwang-noaa** [comment]
@bingfu-NOAA I made a fix in FV3 PR#857. With this fix, you don't need to use fhzero_array/fhzero_fhour. The previous fhzero=6 in input.nml should still work as before. Please let me know if you still have issues. Thanks

**Comment by bingfu-NOAA** [comment]
@junwang-noaa Thanks Jun. We will do some tests and let you know if this update can fix the issue.

**Comment by bingfu-NOAA** [comment]
@junwang-noaa Our tests now show correct bucket after applying this fix.

**Comment by junwang-noaa** [comment]
@bingfu-NOAA Thanks for testing. The code fix is now combined with fv3atm PR#856 and will be committed in UFS WM PR#2355.

**Comment by junwang-noaa** [comment]
The PR was committed in PR#2355. Closing the issue.
